lib/mcstat.inc

```diff
@@ -150,15 +143,15 @@ class mcstat_reporter
   protected $starting_data = null;
   protected $last_data = null;
 
-  function __construct($hostname, $port) {
+  function __construct($hostname, $port, $username, $password) {
```

Comment:
Same here. Should we make these last two parameters optional?

Reply:
Same explanation as above.